src/main/resources/META-INF/plugin.xml

```diff
@@ -11,13 +11,14 @@
 
     <extensions defaultExtensionNs="com.intellij">
         <platform.lsp.serverSupportProvider implementation="com.github.iwanabethatguy.oxcintellijplugin.lsp.OxcLspServerSupportProvider"/>
-        <applicationService serviceImplementation="com.github.iwanabethatguy.oxcintellijplugin.settings.OxcSettingsState"/>
 
-        <applicationConfigurable
+        <projectConfigurable
```

Comment:
project instead of application, so that you can have unique settings for each project you work on.